src/components/widget/ActionButton.js

```diff
@@ -240,6 +270,22 @@ class ActionButton extends Component {
         onBlur={this.handleDropdownBlur}
         onFocus={this.handleDropdownFocus}
       >
+        {promptOpen && (
+          <Prompt
+            title={promptTitle}
+            text={promptText}
+            buttons={{ submit: promptYes, cancel: promptNo }}
+            onCancelClick={() =>
+              this.setState({ promptOpen: false }, () => {
+                this.statusDropdown.blur();
+              })
+            }
+            onSubmitClick={() =>
+              this.processStatus(list.find((elem) => elem.hasOwnProperty('validationInformation')), true)
```

Comment:
hmmm, i have my doubts here, but maybe it's because my lack of knowledge.
I would like to see the dev tests, and mainly see that Completing, Voiding, ReActivating an Sales Order works smooth.

Reply:
and both cases works, when the respective action (DR, VO, CO etc) has some validation information attached and when not.

Reply:
Yeah I would need some more data from the BE in this case from @dragospodariu96  to check.